uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java

```diff
@@ -49,6 +55,7 @@ public static String executeShellCommand(@NonNull final String command) {
     }
 
     public static String executeShellCommandAsCurrentPackage(@NonNull final String command) {
+        Logger.i(TAG, "Execute Shell Command As Current Package..");
```

Comment:
let's log the command in the log message

Reply:
added.